util/timer/timeUntil.js

```
@@ -0,0 +1,20 @@
+module.exports = function(targetTime) {
+    let countDownDate = new Date(targetTime).getTime();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'countDownDate' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)